CleverTapSDK/ios.modulemap

```diff
@@ -9,6 +9,7 @@ framework module CleverTapSDK {
     header "CleverTapEventDetail.h"
     header "CleverTapInAppNotificationDelegate.h"
     header "CleverTapPushNotificationDelegate.h"
+    header "CleverTapURLDelegate.h"
```

Comment:
The user won't be able to import `CleverTapURLDelegate.h` in the Objective-C file since the header is not added in the [podspec](https://github.com/CleverTap/clevertap-ios-sdk/blob/master/CleverTap-iOS-SDK.podspec)